### PR TITLE
add renderHistogram helper and sample family data

### PR DESCRIPTION
**Pull request by jaxonL**
image

### changes
- add `renderHistogram` function (see usage in the relationships section (`main.ts:renderRelationships`)
- add relationship section regarding familial meetings
- ignore temporary generated bundle files and other build files

### other
- sometimes, this error `Error: <rect> attribute width: A negative value is not valid. ("-1")` appears when generating the histogram. It's due to the last element of the generated `bins` object having a width of 0. Some `binCount` values trigger this error, others don't. It does not affect the behaviour of the website. added a quickfix that disregards bins where x1 = x0

### CODE REVIEW

**Review by AkshayPall — APPROVED** [review]
Looks Good! Remember to add this new relationships section to the table of contents `introduction.html:46`. Also please rebase before merging.

**Comment by roxelo** [comment]
LGTM, can't merge yet because of merge conflict :{

**Pull request merged**
